### DISCUSSION TITLE
5주차 반성문

### DISCUSSION THREAD

**Discussion by chairoi** [BDD-CLUB/challenge-every-week-posting#108] [6. 반성문]
반성문마저,.... 늦어버리고 말았습니다... 죄송합니다...
5주차에 이사 준비하고 올라와서 짐 정리하고 하느라 정신이 하나도 없어서.. 사실 컴퓨터에 거의 손도 못 댔습니다
그래도 앞으로 다시 꾸준히 열심히 공부할테니까요
다들 화이팅입니다~~!

**Comment by shkisme** [comment]
대전은,,, 재밋나요!!!! 부럽다!!!!
